src/framework/ui/internal/uiconfiguration.cpp

```diff
@@ -266,15 +266,44 @@ void UiConfiguration::writeThemes(const ThemeList& themes)
     settings()->setValue(UI_THEMES_KEY, value);
 }
 
+ThemeList UiConfiguration::themes() const
+{
+    return m_themes;
+}
+
 QStringList UiConfiguration::possibleFontFamilies() const
 {
     QFontDatabase db;
     return db.families();
 }
 
-ThemeList UiConfiguration::themes() const
-{
-    return m_themes;
+QStringList UiConfiguration::possibleAccentColors() const
+{
+    static const QStringList lightAccentColors {
+        "#F36565",
```

Comment:
Couldn't these be part of the `ThemeInfo` struct? (either directly in the struct or in the `QMap<ThemeStyleKey, QVariant> values`)

Reply:
I think ThemeInfo should contain only values that affect the UI. Those values are used only in the accent colors list and they don't affect to color of any UI element, so better don't store them in the theme